app/controllers/exceptions_controller.rb

```diff
@@ -1,4 +1,4 @@
-class ExceptionsController < ActionController::Base
+class ExceptionsController < ApplicationController
```

Comment:
I think I did this so it would inherit the layout from ApplicationController. I can just add that in here instead. I'm not clear on the repercussions of this change.